internal/service/events/rule.go

```diff
@@ -109,6 +111,30 @@ func ResourceRule() *schema.Resource {
 	}
 }
 
+func jsonMarshal(jsonString interface{}) (string, error) {
```

Comment:
since this is a specific to `event_pattern` we should try to name the function closer to what it is used for.

Can we also add a unit test?